include/librealsense2/h/rs_sensor.h

```diff
@@ -30,6 +30,8 @@ typedef enum rs2_camera_info {
     RS2_CAMERA_INFO_PRODUCT_ID                     , /**< Product ID as reported in the USB descriptor */
     RS2_CAMERA_INFO_CAMERA_LOCKED                  , /**< True iff EEPROM is locked */
     RS2_CAMERA_INFO_USB_TYPE_DESCRIPTOR            , /**< Designated USB specification: USB2/USB3 */
+    RS2_CAMERA_INFO_PRODUCT_LINE                   , /**< Device product line D400/SR300/L500/T200 */
+    RS2_CAMERA_INFO_ASIC_SERIAL_NUMBER             , /**< Asic serial number */
```

Comment:
Should be listed in `rs-enumerate-devices`

Reply:
Isn't it automatically?

Reply:
it's automatic...